packages/story-editor/src/components/library/panes/media/media3p/providerTabList.js

```diff
@@ -108,7 +108,7 @@ function ProviderTabList({ providers }) {
   useKeyDownEffect(ref, ['enter', 'space'], selectFocused, [selectFocused]);
 
   return (
-    <Section ref={ref}>
+    <Section ref={ref} role="tablist">
```

Comment:
children have role of `tab` so this should be a `tablist`

Reply:
Nit: this could be put in the styled component declaration directly

Reply:
sure! updated :)